RollGen.Domain/DomainDice.cs

```diff
@@ -49,7 +49,7 @@ public string ReplaceWrappedExpressions<T>(string source, string expressionOpen
                 var lenientReplacedDice = ReplaceDiceExpression(matchGroupValue, true);
                 var unevaluatedMatch = Evaluate(lenientReplacedDice);
                 var target = expressionOpen + matchGroupValue + expressionClose;
-                var replacement = BooleanOrType<T>(unevaluatedMatch).ToString();
+                var replacement = Utils.BooleanOrType<T>(unevaluatedMatch).ToString();
```

Comment:
A fair note.  Well then, we can leave these as static for now, given the nature of the methods.  I do still want unit tests around them though :)